src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.security.SecureRandom;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        return Color.values()[new SecureRandom().nextInt(Color.values().length)];
```

Comment:
let's use the Random class instead of SecureRandom and make it a constant so that we don't have to create it every time